pkg/artifactory/resource_artifactory_group.go

```diff
@@ -6,7 +6,7 @@ import (
 	"net/http"
 
 	"github.com/atlassian/go-artifactory/v2/artifactory"
-	"github.com/atlassian/go-artifactory/v2/artifactory/v1"
+	v1 "github.com/atlassian/go-artifactory/v2/artifactory/v1"
```

Comment:
Happy to leave these aliased as we already have this in some places and best to keep it consistent, not sure what tool is doing it though.